lib/fluent/plugin/in_tail.rb

```diff
@@ -484,8 +484,8 @@ def close_watcher_handles
     end
 
     # refresh_watchers calls @tails.keys so we don't use stop_watcher -> start_watcher sequence for safety.
-    def update_watcher(target_info, pe)
-      path = target_info.path
+    def update_watcher(watcher, pe, new_inode)
```

Comment:
nit: I prefer tail_watcher for the first argument.
```suggestion
    def update_watcher(tail_watcher, pe, new_inode)
```
In this plugin various type of `watcher`s are appeared so that `watcher`  is confusing.

Probably there are 3 local variable names to represent it in this plugin:

1. `tw`
2. `watcher`
3. `tail_watcher`

1. is also acceptable for me.
Although it's too short but not so confusing in this context, and it's appeared most often in this code.

Reply:
Thanks for your review! I see!
Should we change the method name too?

```diff
- update_watcher(watcher, pe, new_inode)
+ update_tail_watcher(tail_watcher, pe, new_inode)
```

Reply:
Fixed!
I used `tail_watcher`!